services/movie.py

```diff
@@ -1,11 +1,13 @@
+from django.db import transaction
 from django.db.models import QuerySet
 
 from db.models import Movie
 
 
 def get_movies(
-    genres_ids: list[int] = None,
-    actors_ids: list[int] = None,
+        genres_ids: list[int] = None,
+        actors_ids: list[int] = None,
```

Comment:
The type hint `list[int]` requires Python 3.9 or later. Ensure your environment supports this syntax.